### PR TITLE
[deckhouse-config] disable deckhouse-config webhook for uninitialized cluster

### PR DESCRIPTION
**Pull request by diafour**
## Description

- Webhook has stuck in Pending status because it requires initialized cluster
- Deckhouse has stuck on global hook trying to update ConfigMap

 deckhouse-config-webhook seems redundant during cluster bootstrap.

## Why do we need it, and what problem does it solve?

Fix race condition "Global hook can't update the ConfigMap because node-manager can't remove the taint "uninitialized" because global hook holds the 'main' queue."

- Sometimes NodeManager does not have time to remove the taint "uninitialized" and Deckhouse Pod is restarted.
- During startup Deckhouse has stuck on global hook trying to update ConfigMap.
- deckhouse-config-webhook can't tolerate "unintialized" and has stuck in Pending status.

It was observed in real installations (not e2e tests) of Static cluster and in Azure.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: deckhouse-config
type: fix
summary: disable deckhouse-config webhook for uninitialized cluster
impact_level: default
```

### CODE REVIEW

File: modules/003-deckhouse-config/templates/deckhouse-config-webhook/webhook-configuration.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+{{ if .Values.global.clusterIsBootstrapped }}
```
Comment (yalosev) [inline modules/003-deckhouse-config/templates/deckhouse-config-webhook/webhook-configuration.yaml:1]:
```suggestion
{{- if .Values.global.clusterIsBootstrapped }}
---
```

File: modules/003-deckhouse-config/templates/deckhouse-config-webhook/webhook-configuration.yaml [hunk 2]
```diff
@@ -54,3 +55,4 @@ webhooks:
         name: deckhouse-config-webhook
         namespace: d8-system
         path: /validate-cm
+{{ end }}
```
Comment (yalosev) [inline modules/003-deckhouse-config/templates/deckhouse-config-webhook/webhook-configuration.yaml:58]:
```suggestion
{{- end }}
```

File: modules/003-deckhouse-config/templates/deckhouse-config-webhook/webhook-configuration.yaml (hunk 1 above)
Comment (diafour) [reply modules/003-deckhouse-config/templates/deckhouse-config-webhook/webhook-configuration.yaml]: Done

**Review by yalosev — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [3274](https://github.com/deckhouse/deckhouse/pull/3274) to the branch [release-1.41](https://github.com/deckhouse/deckhouse/tree/release-1.41) successful!

**Pull request merged**